x-pack/test/cases_api_integration/security_and_spaces/tests/common/connectors/case.ts

```diff
@@ -5,37 +5,13 @@
  * 2.0.
  */
 
-import { omit } from 'lodash/fp';
-import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
 
-import { CASES_URL } from '../../../../../../plugins/cases/common/constants';
-import { CommentType } from '../../../../../../plugins/cases/common/api';
-import { postCaseReq, postCaseResp } from '../../../../common/lib/mock';
-import {
-  removeServerGeneratedPropertiesFromCase,
-  removeServerGeneratedPropertiesFromComments,
-} from '../../../../common/lib/utils';
-import {
-  createRule,
-  createSignalsIndex,
-  deleteAllAlerts,
-  deleteSignalsIndex,
-  getRuleForSignalTesting,
-  getSignalsByIds,
-  waitForRuleSuccessOrStatus,
-  waitForSignalsToBePresent,
-} from '../../../../../detection_engine_api_integration/utils';
-
 // eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext): void => {
   const supertest = getService('supertest');
-  const esArchiver = getService('esArchiver');
-  const log = getService('log');
 
   describe('case_connector', () => {
-    let createdActionId = '';
-
     it('should return 400 when creating a case action', async () => {
```

Comment:
I think we should delete the file. What do you think?

Reply:
Yep, good catch